clients/drcachesim/tests/offline-skip.expect

```diff
@@ -1,16 +1,16 @@
 Output format:
 <--record#-> <--instr#->: <---tid---> <record details>
 ------------------------------------------------------------
-         114          62:     1992554 <marker: timestamp 13352268558646596>
-         114          62:     1992554 <marker: tid 1992554 on core 10>
-         115          63:     1992554 ifetch       2 byte(s) @ 0x0000000000401026 0f 05                syscall  -> %rcx %r11
-         116          63:     1992554 <marker: timestamp 13352268558646600>
-         117          63:     1992554 <marker: tid 1992554 on core 10>
-         118          63:     1992554 <marker: system call 1>
-         119          63:     1992554 <marker: maybe-blocking system call>
-         120          63:     1992554 <marker: timestamp 13352268558646604>
-         121          63:     1992554 <marker: tid 1992554 on core 10>
-         122          64:     1992554 ifetch       4 byte(s) @ 0x0000000000401028 48 83 eb 01          sub    $0x0000000000000001 %rbx -> %rbx
+         114          62:      1e676a <marker: timestamp 13352268558646596>
+         114          62:      1e676a <marker: tid 0x1e676a on core 10>
```

Comment:
So we expect users to know that some MSBs represent the workload. In that case, let's give an example for how the workload id and tid should be extracted, by adding that logic to the view tool.

It would be better if the view tool printed: 

`<marker: workload 0x1 tid 0xe676a on core 10>`

Reply:
Changed the view tool to print the `W0.T1234` form that schedule_stats was using.